src/app/features/psbt-signer/components/psbt-request-details/components/psbt-request-fee.tsx

```diff
@@ -0,0 +1,32 @@
+import { Stack, Text, color } from '@stacks/ui';
+
+import { Money } from '@shared/models/money.model';
+
+import { formatMoney, i18nFormatCurrency } from '@app/common/money/format-money';
+import { SpaceBetween } from '@app/components/layout/space-between';
+import { useCalculateBitcoinFiatValue } from '@app/query/common/market-data/market-data.hooks';
+
+import { PsbtRequestDetailsSectionLayout } from './psbt-request-details-section.layout';
+
+export function PsbtRequestFee(props: { fee: Money }) {
+  const { fee } = props;
+  const calculateBitcoinFiatValue = useCalculateBitcoinFiatValue();
+
+  return (
+    <PsbtRequestDetailsSectionLayout>
+      <SpaceBetween>
+        <Text fontSize={2} fontWeight="500">
```

Comment:
We could extract it to it's own component to avoid repeating.

In general throughout the APP we have a lot of repeated text and styles. When implementing the new design system we should consider:
- implementing tokens to reuse for styles
- extracting text to `i18n` / config files